src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.FluentWindows/Styles/Calendar.xaml

```diff
@@ -353,7 +353,7 @@
                                     <Button.Content>
                                         <TextBlock
                                             FontSize="8"
-                                            FontFamily="Segoe Fluent Icons"
+                                            FontFamily="{DynamicResource SegoeFluentIcons}"
```

Comment:
Can all these be `StaticResource` instead? I don't see a case where `SegoeFluentIcons` would have any other value.

Reply:
Also, how are we aiming to change the font based on system if that is the case? Shouldn't we define this font value somewhere in a common file and update that value based on system value and use the defined value as a `DynamicResource`. Is this under the scope of these changes?